frontend/components/site/SiteSettings/BasicSiteSettings.jsx

```diff
@@ -83,7 +91,7 @@ export const BasicSiteSettings = ({
               label="Demo URL:"
               name="demoDomain"
               id="demoDomainInput"
-              placeholder="https://preview.example.gov"
+              placeholder="https://demo.example.gov"
```

Comment:
changed to `demo` to both match the prose and so we aren't using the word `preview` in relation to the demo site